apis/Google.Cloud.Logging.Console/SampleApp/Program.cs

```diff
@@ -18,8 +18,19 @@
 
 var builder = WebApplication.CreateBuilder(args);
 
+var config = new ConfigurationBuilder()
```

Comment:
I don't think you need to do this: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0.

Then you use the [builder.Configuration](https://learn.microsoft.com/en-gb/dotnet/api/microsoft.aspnetcore.builder.webapplicationbuilder.configuration?view=aspnetcore-6.0) property to access the config sections you need.